.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
@alexleventer does it make sense to add README instructions about installing husky and pre-commit? Otherwise the lint pre-commit may not run at all or fail.

```
...
git add .husky/pre-commit
```